dissect/target/plugin.py

```diff
@@ -528,6 +534,7 @@ def register(plugincls: type[Plugin]) -> None:
                         exported=exported,
                         internal=internal,
                         findable=plugincls.__findable__,
+                        alias=getattr(attr, "__alias__", False),
```

Comment:
```suggestion
                        alias=name in getattr(attr, "__aliases__", []),
```
I think we can do this without introducing another attribute on the function itself.